sdk/src/store.rs

```diff
@@ -186,6 +186,20 @@ impl Store {
         Ok(claim_label)
     }
 
+    /// Add a new Claim to this Store. [`remote_manifest_path`] will be embedded in the
+    /// final asset once saved.  If [`remote_manifest_path`] is None an external side car file
+    /// will be generated instead. The function will return the label of the claim.
+    pub fn commit_claim_external(
```

Comment:
Why do we need this vs just calling claim.set_remote_manifest prior to calling commit_claim?

Reply:
It is the Rust way.  Just setting a field param does not really indicate the side affects.  The different commit makes it more clear.   I am not attached this either way.

Reply:
The Option parameter None isn't very Rusty either. It doesn't describe adding a sidecar. It would make more sense taking a RemoteManifest parameter without the option. I'd argue that there a lot of things in a claim that can change what the commit operation does.